db/functions/base.py

```diff
@@ -86,6 +86,19 @@ def to_sa_expression(primitive):
         return literal(primitive)
 
 
+class Noop(DBFunction):
```

Comment:
Since `Noop` is so similar to `Literal`, could you explain in a docstring what it is necessary? I presume it's because `Literal` creates an SQLAlchemy expression like `literal(subexpression)`, which in some way doesn't play well with the subexpression being a list of literals? Thanks!

Reply:
Thanks for your review! Since we are using array as a input in `JsonArrayContains`, it is having trouble passing the filtering tests because of the literal() function. Thus, we need another function without literal wrapping to have that porperly implemented.

Reply:
@Jinxiao0302 Given the confusion, I do think it makes sense to add a docstring indicating the reason for this DBFunction.
- The `Literal` DB function produces a `literal` SQLAlchemy wrapper which doesn't play nicely with the type conversion between python classes and db types in `psycopg2`. 

@dmos62 Long-term, we need to reassess how we're dealing with literal (in the general sense) values in the DBFunction subclasses. Wrapping an arg in `literal` should probably be avoided, except in situations where we actually want to produce a column of literals in the produced query. I don't think it should ever be used for the comparand in a filter.